src/mixins/themeable.ts

```diff
@@ -7,22 +7,34 @@ import { ComposeFactory } from '@dojo/compose/compose';
 import { assign } from '@dojo/core/lang';
 
 /**
- * A representation of the css-module class names
- * to be applied where each class in appliedClasses
- * is used.
+ * A representation of the css-module class names to be applied
+ * where each class in appliedClasses is used.
  */
-export type CSSModuleClassNames = {
+export type AppliedCSSModuleClassNames = {
 	[key: string]: boolean;
 }
 
 /**
- * The object returned by getClasses.
+ * A mapping from class names to generatred css-module class names
  */
-export type AppliedClasses<T> = {
-	[P in keyof T]: CSSModuleClassNames;
-};
+export type CSSModuleClassNameMap = {
```

Comment:
How about `GeneratedClassNameMap`? Is there any reason why this shouldn't be a `Map`?